### PR TITLE
Make definition of substantive changes more specific

### PR DESCRIPTION
**Pull request by justinwb**
This pull request is aimed to solve the points raised by @RubenVerborgh in solid/culture#57, where the language in the current process that determines whether something is a substantive change, and how to proceed if it is (or not), needed further clarity.

### CODE REVIEW

**Comment by Mitzi-Laszlo** [comment]
Re: Candidate Proposals without substantive changes require one editor to actively approve the proposal with no editors actively rejecting. Editors may abstain. Candidate Proposals without substantive changes must remain open for at least twenty-four hours before final acceptance. If an editor does not vote by the end of that twenty-four hour period it will be assumed that the editor abstained.

* The open time needs to be a little longer to give an editor the chance to agree that it is not substantive. It is quite possible that an editor may not check GitHub for 24 hours. Perhaps a week is more reasonable? 
	
Once a Candidate Proposal has successfully passed editorial review and the specified wait-time has elapsed, it may be merged by an editor, or by one of its authors if they have appropriate permission to do so.

* Isn't it the role of admins to merge the result of editors? Perhaps best to maintain consistency in the division of voting and executing?

**Comment by justinwb** [comment]
> * The open time needs to be a little longer to give an editor the chance to agree that it is not substantive. It is quite possible that an editor may not check GitHub for 24 hours. Perhaps a week is more reasonable?

Perhaps one day is too fast of a turnaround, but I think one week for a non-substantive change (for example, to fix a typo), is too long. That's the same amount of review time we ask for a substantive change. I don't think they should be equivalent. Maybe two days is more appropriate?

> * Isn't it the role of admins to merge the result of editors? Perhaps best to maintain consistency in the division of voting and executing?

If something has already passed editorial review I don't think there's a real reason to mandate that only Administrators can merge it. Adding people into the mix that aren't directly involved in the editorial review process adds another step, and I'm not sure what value that brings. If someone is an approved editor, or has authored something good enough to pass editorial review, they should be allowed to merge it on their own if they have appropriate permissions already to do so.

**Comment by TallTed** [comment]
It's important to remember that part of the point of git (and similar) is to allow *reversion of changes* -- so if something is merged (by an editor or whomever) as non-substantive and subsequently determined (by the director or whomever) to be substantive, that merge can be reverted (usually easily if not trivially; *occasionally* with some difficulty, if subsequent changes were based on it).

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Anyone may propose improvements to the Solid Specification, Solid Roadmap, or Su
 - Make a suggestion on the Solid W3C Community Group mailing list to form a new Solid Panel, or join an existing Solid Panel
 - Propose an item for the W3C Solid Community Group call
 
-Proposals for substantive changes to the Solid Specification, Solid Roadmap, or Supporting Documentation go through an editorial review process. A change is considered substantive when it alters the normative text of the Solid Specification or the Solid Roadmap.
+Proposals for changes to the Solid Specification, Solid Roadmap, or Supporting Documentation go through an editorial review process. Proposals that include substantive changes which significantly alter the normative text of the Solid Specification or the Solid Roadmap go through a more comprehensive editorial review.
```
Comment (RubenVerborgh) [inline README.md]: I would remove "normative".

File: README.md [hunk 2]
```diff
@@ -64,9 +64,15 @@ When there are objections, notify the Solid Panels and Stakeholders about the fi
 
 ### Reviewing proposals
 
-Candidate proposals to the Solid Specification, Solid Roadmap, or Supporting Documentation submitted for review go through an editorial process before they are accepted.
+Candidate Proposals to change the Solid Specification, Solid Roadmap, or Supporting Documentation must be submitted for editorial review before they are accepted, along with the results of any votes taken while the proposal was being drafted.
```
Comment (RubenVerborgh) [inline README.md]: There might be a conflict with the edited paragraph above; maybe the paragraph above can just be removed, as its meaning seems to be in here?

File: README.md [hunk 3]
```diff
@@ -64,9 +64,15 @@ When there are objections, notify the Solid Panels and Stakeholders about the fi
...
 
-Candidate proposals and the results of any vote related to the same need to be put forward to the editors for review. For the proposal to be accepted, three editors need to actively approve with no editors actively rejecting. Editors may abstain. A proposal must be open for at least one week before final acceptance. If an editor does not vote by the end of that week it will be assumed that the editor abstained.
+An editor determines whether a Candidate Proposal includes a substantive change and marks it accordingly. If there is any disagreement among editors, the Candidate Proposal will be automatically marked as including a substantive change.
```
Comment (RubenVerborgh) [inline README.md]: 👍 That's the kind of trust we need in people. We should trust that editors can decide themselves what is substantive and what is not.

File: README.md (hunk 1 above)
Comment (RubenVerborgh) [reply README.md]: On second thought, I think this entire paragraph can go (see below).

File: README.md [hunk 4]
```diff
@@ -64,9 +64,15 @@ When there are objections, notify the Solid Panels and Stakeholders about the fi
...
+
+Candidate Proposals with substantive changes require three editors to actively approve the proposal with no editors actively rejecting. Editors may abstain. Candidate Proposals with substantive changes must remain open for at least one week before final acceptance. If an editor does not vote by the end of that week it will be assumed that the editor abstained.
+
+Candidate Proposals without substantive changes require one editor to actively approve the proposal with no editors actively rejecting. Editors may abstain. Candidate Proposals without substantive changes must remain open for at least twenty-four hours before final acceptance. If an editor does not vote by the end of that twenty-four hour period it will be assumed that the editor abstained.
```
Comment (RubenVerborgh) [inline README.md]: I don't think the 24-hour period is needed. Could just be typos etc., would be massive slowdown.

**Comment by kjetilk** [comment]
To elaborate on @TallTed 's point of reverts, it enables us to live with relatively short time periods for non-substantiative changes. I think 24h is the longest acceptable, but that any editor could open a proposal to revert.

Any editor may also call for a revert on something that was deemed a non-substantiative change to upgrade it to a substantiative change.

**Comment by justinwb** [comment]
> Any editor may also call for a revert on something that was deemed a non-substantiative change to upgrade it to a substantiative change.

Great points. I'm happy removing the time period altogether and adding in some language about the ability for editors to revert non-substantive changes. Anyone have a fundamental disagreement with that?

File: README.md (hunk 1 above)
Comment (justinwb) [reply README.md]: Agree - going to just remove this one altogether 👍

File: README.md (hunk 4 above)
Comment (justinwb) [reply README.md]: Agree - will remove the time period and point out ability to revert non-substantive.

**Comment by justinwb** [comment]
This pull request has been succeeded by #95. This branch and its associated version history was merged into that branch and then modified further. Closing this pull request.

**Pull request closed without merging**